### ISSUE TITLE
Kubectl location preferences are lost on every launch of Lens ide

### ISSUE THREAD

**Issue by aaraodeo** [lensapp/lens#1010]
**Describe the bug**
Kubectl location preferences are lost on every launch of Lens ide

**To Reproduce**
1. Start Lens ide
2. Last configured cluster is opened.
3. Able to see pods in workloads
4. Open Terminal. An Error Alert message is displayed:
   A JavaScript error occurred  in the main process.
   Uncaught Exception:
   Error:  EPERM: operation not permitted , open
   'C:\Users\user\AppData\Roaming\Lens\binaries\kubectl\1.18.8\kubectl.exe'
5. Close the Alert message.
6. Goto File >> Preferences, unCheck Kubectl Binary
   In path to Kubectl binaries, specify
   C:\Program Files\Docker\Docker\resources\bin\kubectl.exe
7. Close Preferences
8. Open Terminal, issue following command:
   kubectl version --short
  It works and gives following output:
  Client Version: v1.18.8
  Server Version: v1.18.8
9. Close the Lens Ide
10. Start the Lens Ide again
11. The issue re-occurrs, i have to set again the kubectl path

**Expected behavior**
Kubectl location preferences needs to be preserved for user.

**Screenshots**
N/A

**Environment (please complete the following information):**
- Lens Version: 3.6.5
- OS: Windows 10 pro 64 bit 1909 (OS Build 18363.1082)
- Installation method (e.g. snap or AppImage in Linux): 
 Lens-Setup-3.6.5.exe on Windows 10 OS
https://github.com/lensapp/lens/releases/download/v3.6.5/Lens-Setup-3.6.5.exe
**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
N/A
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: DATA+OMITTED
    server: https://kubernetes.docker.internal:6443
  name: docker-desktop
contexts:
- context:
    cluster: docker-desktop
    user: docker-desktop
  name: docker-desktop
current-context: docker-desktop
kind: Config
preferences: {}
users:
- name: docker-desktop
  user:
    client-certificate-data: REDACTED
    client-key-data: REDACTED
```

**Additional context**
Testing on Docker Desktop Version: 2.4.0

**Comment by Nokel81** [comment]
I have attempted to reproduce this. (Though I am on macos which might make a difference). I wonder if I could get a screen shot of the Lens Preferences page?

Having tested this on macos, it seems to be working as intended (namely, with the "Download kubectl binaries" setting as `false` the provided kubectl path is used on startup)

**Comment by nevalla** [comment]
Can't reproduce this on Windows either. 

Noticed that you've mentioned **AppImage on Windows 10** as installation method. Is that really so?

**Comment by nevalla** [comment]
Also you don't necessary need to switch off kubectl downloading, you can define another directory where Lens will download kubectl binaries based on cluster versions.

**Comment by aaraodeo** [comment]
My mistake i mentioned earlier i installed using Appimage on windows, not correct. I installed using https://github.com/lensapp/lens/releases/download/v3.6.5/Lens-Setup-3.6.5.exe on windows 10 pro 64 bit os.

Its incorrect on lens ide part if it always try to download kubectl binary instead of using the existing one like the one provided as part of docker-desktop.

Also it must auto detect if permission is there to save file in the specified folder and ask user to provide a different folder instead of throwing an error alert to user every time a new terminal is launched.

Every time i launch lens ide, its detect last configured cluster and opens it correctly. I can view pods in workloads.
But as soon as i open the terminal i get the error alert as mentioned in issue description at top.

To resolve this, I navigate to File >> Preferences, unCheck Kubectl Binary
In path to Kubectl binaries, specify
C:\Program Files\Docker\Docker\resources\bin\kubectl.exe, then close preferences.
Then if i open terminal it opens without any error and i am able to run kubectl version --short command and get the expected output.

I had observed this issue since last couple of lens ide releases. I have to repeat the steps everytime i launch the lens ide.

What needs to be done is auto detect if docker-desktop is being used and set the default kubectl path to C:\Program Files\Docker\Docker\resources\bin\kubectl.exe and do not download any other kubectl binary, which may be incompatible with docker-desktop.

Also i do not get the purpose of File >> Preferences >> Helm >> Repositories
It always shows "No Options"
While i have couple of repositories added in helm which are not shown in lens ide.
Ex:-
helm repo list
NAME            URL
stable          https://kubernetes-charts.storage.googleapis.com/
ingress-nginx   https://kubernetes.github.io/ingress-nginx/
I am using helm v3.3.4
